Makefile

```diff
@@ -1,113 +1,26 @@
-ifeq ($(BOLOS_SDK),)
-$(error Environment variable BOLOS_SDK is not set)
-endif
+#*******************************************************************************
+#   Ledger App
+#   (c) 2024 Ledger
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#*******************************************************************************
 
-include $(BOLOS_SDK)/Makefile.defines
-
-########################################
-#        Mandatory configuration       #
-########################################
-# Application name
 APPNAME = "P2P.org"
+NORMAL_NAME = "p2p"
```

Comment:
What is the purpose of this `NORMAL_NAME` ?

Reply:
It's to set the variant as 'p2p' instead of derived from appname 'p2p_org'